### DISCUSSION TITLE
Epic games and modding

### DISCUSSION THREAD

**Discussion by happysong123** [diesel-modding/PAYDAY-2-BeardLib#666] [Q&A] [answered]
As you may know Payday2 is now on Epic games store...however  Beard lib is not working as one of the mods there...I installed it " Payday2 on epic games " and when it comes to modding the game it always crashed out when i install BeardLib, i hope you make an update for it on this platfrom. 
Thank you.

**Comment by segabl** [comment, marked as answer]
You need to download the most recent version of BeardLib from here, the crashes on Epic should have been fully fixed.

**Reply by happysong123** [reply]
Yeah it worked Thanks dude
